src/js/experimental/media-chrome-option.js

```diff
@@ -13,17 +13,21 @@ template.innerHTML = /*html*/`
     min-height: 1.2em;
     padding: .4em .5em;
     transition: var(--media-option-transition);
+    outline: var(--media-option-outline, 0);
+    outline-offset: var(--media-option-outline-offset, -1px);
   }
 
   :host(:focus-visible) {
-    box-shadow: inset 0 0 0 2px rgb(27 127 204 / .9);
-    outline: 0;
+    box-shadow: var(--media-option-focus-shadow, inset 0 0 0 2px rgb(27 127 204 / .9));
+    outline: var(--media-option-hover-outline, 0);
+    outline-offset: var(--media-option-hover-outline-offset, -1px);
```

Comment:
can this be without an explicit -1px as default? 

like can it get it from `:host` by default?

Reply:
do you mean add another var higher up the chain that it inherits from? (that itself is defaulted to -1px?)

Reply:
yea I was thinking it should get the default automatically from 

```css
:host { outline-offset: var(--media-option-outline-offset, -1px); }
```

not sure if it needs an explicit `revert` or `inherit` or so 

could also do something like if it's not easy with CSS keywords

`outline-offset: var(--media-option-hover-outline-offset, var(--media-option-outline-offset, -1px))`

Reply:
`revert` and `inherit` doesn't seem to work, assumedly because it's not by default an inheritable property. But I've made the hover states use the non-hover offset property as a fallback like in your last example.